go/vt/schemadiff/table.go

```diff
@@ -724,6 +750,10 @@ func (c *CreateTableEntity) TableDiff(other *CreateTableEntity, hints *DiffHints
 	alterTable := &sqlparser.AlterTable{
 		Table: c.CreateTable.Table,
 	}
+	if hints.TableQualifierHint == TableQualifierDeclared {
+		alterTable.Table.Qualifier = other.Table.Qualifier
+	}
+
```

Comment:
should this only apply to `AlterTableDiff` or also to `AlterViewDiff`? And to `CREATE/DROP` as well? etc. etc.?